ProjectSettings/ProjectSettings.asset

```diff
@@ -670,6 +670,6 @@ PlayerSettings:
   projectName: 
   organizationId: 
   cloudEnabled: 0
-  enableNativePlatformBackendsForNewInputSystem: 0
-  disableOldInputManagerSupport: 0
+  enableNativePlatformBackendsForNewInputSystem: 1
+  disableOldInputManagerSupport: 1
```

Comment:
Are we sure that we are going to use the new input system instead of the default one?